.github/autobump.txt

```diff
@@ -1723,6 +1723,7 @@ trino
 trivy
 trojan-go
 trufflehog
+tsduck
```

Comment:
Didn't follow the discussion about adding this to the autobump list, so feel free to open a new PR to remove it if it's not suitable.

Reply:
Thanks @carlocab, it is fine with me.

I am not familiar with autobump. My understanding is that the brew bot will automatically detect a new "version" (tag? release?) of the upstream project on GitHub and will autobump in HomeBrew without need to submit a PR. Is this correct?